index2.html

```diff
@@ -0,0 +1,177 @@
+<!doctype html>
```

Comment:
Generally speaking it's better to let git do it's thing and remove old files :) But I can see why you did it with the major rewrite

Reply:
Yeah inefficiency due to git inexperience